src/pubtools/_marketplacesvm/tasks/community_push/command.py

```diff
@@ -261,9 +261,10 @@ def set_accounts(acct_name: str, acct_dict: Dict[str, List[str]]) -> None:
             log.info("Loading %s from StArMap: %s", acct_name, accts)
             if isinstance(accts, dict):
                 combined_accts = []
-                for _, accounts in accts.items():  # expected format: Dict[str, List[str]]
-                    combined_accts.extend(accounts)
-                    log.debug("Loaded \"%s\": \"%s\" from StArMap.", acct_name, accounts)
+                for _, accounts in accts.items():  # expected format: Dict[str, str]]
```

Comment:
accounts -> account :-)
There is not a list anymore

Reply:
Adjusted